docs/conf.py

```diff
@@ -0,0 +1,217 @@
+
+# List of patterns, relative to source directory, that match files and
+# directories to ignore when looking for source files.
+# This pattern also affects html_static_path and html_extra_path.
+exclude_patterns = [
+    "spelling_wordlist.txt",
+    "**/CHANGES.rst",
+    "**/LICENSE.rst",
+]
+
+
+# -- Options for myST markdown conversion to html -----------------------------
+
+# For more information see:
+# https://myst-parser.readthedocs.io/en/latest/syntax/optional.html
+myst_enable_extensions = [
+    "deflist",  # You will be able to utilise definition lists
+                # https://myst-parser.readthedocs.io/en/latest/syntax/optional.html#definition-lists
+    "linkify",  # Identify “bare” web URLs and add hyperlinks.
+    "colon_fence",  # You can also use ::: delimiters to denote code fences,\
+                    #  instead of ```.
+]
+
+
+# -- Intersphinx configuration ----------------------------------
+
+# This extension can generate automatic links to the documentation of objects
+# in other projects. Usage is simple: whenever Sphinx encounters a
+# cross-reference that has no matching target in the current documentation set,
+# it looks for targets in the documentation sets configured in
+# intersphinx_mapping. A reference like :py:class:`zipfile.ZipFile` can then
+# linkto the Python documentation for the ZipFile class, without you having to
+# specify where it is located exactly.
+#
+# https://www.sphinx-doc.org/en/master/usage/extensions/intersphinx.html
+#
+intersphinx_mapping = {
+    "training": ("https://training.plone.org/5/", None),
+    "python": ("https://docs.python.org/3/", None),
+}
+
+
+# -- GraphViz configuration ----------------------------------
+
+graphviz_output_format = "svg"
+
+
+# -- OpenGraph configuration ----------------------------------
+
+ogp_site_url = "https://training.plone.org/5/"
+ogp_description_length = 200
+ogp_image = "https://docs.plone.org/_static/Plone_logo_square.png"
+ogp_site_name = "Plone Documentation"
+ogp_type = "website"
+ogp_custom_meta_tags = [
+    '<meta property="og:locale" content="en_US" />',
+]
+
+
+# -- sphinx_copybutton -----------------------
+copybutton_prompt_text = r"^ {0,2}\d{1,3}"
+copybutton_prompt_is_regexp = True
+
+
+# -- Options for HTML output -------------------------------------------------
+
+# The theme to use for HTML and HTML Help pages.  See the documentation for
+# a list of builtin themes.
+#
+html_theme = "sphinx_book_theme"
+
+html_logo = "_static/logo.svg"
+html_favicon = "_static/favicon.ico"
+
+html_css_files = ["custom.css",
+                  ("print.css", {"media": "print"})]
+
+# See http://sphinx-doc.org/ext/todo.html#confval-todo_include_todos
+todo_include_todos = True
+
+# Announce that we have an opensearch plugin
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-html_use_opensearch
+html_use_opensearch = "https://docs.plone.org"
+
+html_theme_options = {
+    "google_analytics_id": "UA-1907133-6",
```

Comment:
Is the Google Analytics Tracking-Code for dev different from Tracking-Code of docs.plone.org?

Reply:
No, it is the same code.

Should I use a different one?

How do I get a different one?

Reply:
To not clutter the account of live documentation a different code would be good. Or maybe removing from dev and opening a reminding issue to bring it back on relaunch is sufficient.

Reply:
Done in 95404a22c142a3e7a5dc36c34e10755583f7df89 and https://github.com/plone/documentation/issues/1158.